openslides/core/config_variables.py

```diff
@@ -0,0 +1,153 @@
+from django.core.validators import MaxLengthValidator
+from django.utils.translation import ugettext as _
+from django.utils.translation import ugettext_lazy
+
+from openslides.core.config import ConfigVariable
+
+
+def get_config_variables():
+    yield ConfigVariable(
```

Comment:
Docstring: `Generator which yields all config variables of this app. There are two main groups: 'General' and 'Projector'. The group 'General' has subgroups. The generator has to be evaluated during app loading (see apps.py).`